tools/c7n_azure/tests/test_app_service_plan.py

```diff
@@ -78,6 +78,45 @@ def test_resize_plan(self):
         resources = p.run()
         self.assertEqual(len(resources), 1)
 
+    @arm_template('appserviceplan.json')
+    def test_resize_plan_from_resource_tag(self):
+        p1 = self.load_policy({
+            'name': 'test-azure-appserviceplan',
+            'resource': 'azure.appserviceplan',
+            'filters': [
+                {'type': 'value',
+                 'key': 'name',
+                 'op': 'eq',
+                 'value_type': 'normalize',
+                 'value': 'cctest-appserviceplan'}],
+            'actions': [
+                {'type': 'tag',
+                 'tag': 'sku',
+                 'value': 'B1'}]
+        })
+
+        p1.run()
```

Comment:
I think this is better to use SDK client instead of c7n to apply tag. Reason: action is async, so there is no guarantee tag is available when we run second policy.